src/WebJobs.Script.WebHost/Configuration/HostEasyAuthOptionsSetup.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
+{
+    public class HostEasyAuthOptionsSetup : IConfigureOptions<HostEasyAuthOptions>
+    {
```

Comment:
Add unit tests. See https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests/Configuration/HttpWorkerOptionsSetupTests.cs for an example or any tests in that folder.